core/frontend/src/main/scala/io/udash/core/Definitions.scala

```diff
@@ -68,8 +68,8 @@ trait View {
 
 /** A [[io.udash.core.View]] which does not have any child view. */
 trait FinalView extends View {
```

Comment:
Since we're already making major changes, it might be cleaner to just separate the final and non-final views as equal inheritors of View instead of making them a special case handled with an exception. This would move `renderChild` to a meaningful trait for non-final views.